### DISCUSSION TITLE
dirtyFields  with their values

### DISCUSSION THREAD

**Discussion by Ali-Hussein-dev** [react-hook-form/react-hook-form#7236] [Q&A] [answered]
Is there a way to get modified fields values not just their names? or have I to find my workaround? I searched the docs but couldn't find any.

**Comment by fahadsohail482** [comment, marked as answer]
@Ali-Hussein-dev Extract all the fields from dirtyFields object and pass those to an array like so `getValues(["dirtyField1","dirtyField2"])`

This [codesandbox](https://codesandbox.io/s/muddy-leaf-y8y3g?file=/src/App.js) might give you an idea on how to do that.

**Comment by Ali-Hussein-dev** [comment]
@fahadsohail482 great thank you!
